doc/overview.rst

```diff
@@ -37,13 +37,14 @@ the crowd:
    great choice for multi-project development.
 
 -  Cross-platform: as long as your code is portable, Dune will be
-   able to cross-compile it (note that Dune is designed internally
-   to make this easy, but the actual support is not implemented yet)
+   able to cross-compile it. Dune is designed internally
+   to make this easy. Read more in the [Cross-Compilation Doc](https://dune.readthedocs.io/en/stable/cross-compilation.html)
```

Comment:
Rather than linking, it should be possible to use `:ref:section` here. You will need to put a `section` label in the cross-compilation rst file.

Reply:
Thanks! I think I did this right. :)